src/components/AppLayout/MobileNotSupported/index.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { ReactElement } from 'react'
+import styled from 'styled-components'
+import { Button, Text, Card, Icon } from '@gnosis.pm/safe-react-components'
+import { MobileView } from 'react-device-detect'
+import { rgba } from 'polished'
```

Comment:
@alongoni `polished` is not longer a project dependency, it was removed a few versions ago. To use opacity function please use material-ui one as we did here:

https://github.com/gnosis/safe-react/pull/1572/files#diff-18087704d677bfdd0a0bcc5077965f539199fd68aae258e29265f150f883aec9L2